Code example for Pubsub component using Dapr Java SDK (#529)
dapr/quickstarts: pub_sub/java/sdk/README.md

# Dapr pub/sub

> In this quickstart, you'll create a publisher microservice and a subscriber microservice to demonstrate how Dapr enables a publish-subcribe pattern. The publisher will generate messages of a specific topic, while subscribers will listen for messages of specific topics. See [Why Pub-Sub](#why-pub-sub) to understand when this pattern might be a good choice for your software architecture.

Reviewer: ```suggestion
In this quickstart, you'll create a publisher microservice and a subscriber microservice that demonstrates how Dapr enables a publish-subscribe pattern. The publisher will publish messages to a specific topic, while subscribers will listen for messages from a specific topic. See [Why Pub-Sub](#why-pub-sub) to understand when this pattern might be a good choice for your software architecture.
```

Author: We have similar wordings at other examples, So to keep it consistent I have used similar wordings.

Visit [this](https://docs.dapr.io/developing-applications/building-blocks/pubsub/) link for more information about Dapr and Pub-Sub.

> **Note:** This example leverages HTTP `requests` only.  If you are looking for the example using the Dapr Client SDK (recommended) [click here](../sdk/).

This quickstart includes one publisher:

- Java client message generator `checkout`

And one subscriber:

- Java subscriber `order-processor`

## Pre-requisites

* [Dapr and Dapr Cli](https://docs.dapr.io/getting-started/install-dapr/).

> * Java JDK 11 (or greater): [Oracle JDK](https://www.oracle.com/technetwork/java/javase/downloads/index.html#JDK11) or [OpenJDK](https://jdk.java.net/13/).

Reviewer: provide the same version for openjdk

* [Apache Maven](https://maven.apache.org/install.html) version 3.x.

> ### Run Java message publisher app with Dapr

Reviewer: blank line above.
